sdk/communication/azure-communication-email/CHANGELOG.md

```diff
@@ -19,10 +19,6 @@
 - Custom headers in the email message are now key/value pairs.
 - The importance property was removed. Email importance can now be specified through either the `x-priority` or `x-msmail-priority` custom headers.
```

Comment:
Similar comment here about mentioning LRO here.

Reply:
Listed under the "Breaking Changes" section